sentry-android-core/src/test/java/io/sentry/android/core/SentryPerformanceProviderTest.kt

```diff
@@ -0,0 +1,37 @@
+package io.sentry.android.core
+
+import android.content.pm.ProviderInfo
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import kotlin.test.BeforeTest
+import kotlin.test.Test
+import kotlin.test.assertNotNull
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class SentryPerformanceProviderTest {
+
+    @BeforeTest
+    fun `set up`() {
+        AppStartState.getInstance().resetInstance()
+    }
+
+    @Test
+    fun `provider sets app start`() {
+        val providerInfo = ProviderInfo()
+
+        val mockContext = ContextUtilsTest.createMockContext()
+        providerInfo.authority = AUTHORITY
+
+        val provider = SentryPerformanceProvider()
+        provider.attachInfo(mockContext, providerInfo)
+
+        // done by ActivityLifecycleIntegration so forcing it here
+        AppStartState.getInstance().setAppStartEnd()
+
+        assertNotNull(AppStartState.getInstance().appStartInterval)
```

Comment:
`l`: We could assert for an expected interval and not just assert if it's not null.

Reply:
it'd not be possible to mock `appStartMillis` and `appStartTime` anyway because we want to get the time during class load which is before we can inject something via ctor, if we do via ctor, we are losing some millis, that's why just testing if its at least set